src/simulate/main.py

```diff
@@ -60,28 +61,52 @@ def main():
             sys.exit(1)
 
     files = os.listdir(json_files_dir)
+    session = requests.session()
     logging.info('list of json files in the directory %s', files)
+    event_count = 0
     for i in range(run_count):
         for file in files:
             if file.endswith(".json"):
                 path_to_file = json_files_dir + "/" + file
```

Comment:
This whole section chosen by the ".json" check needs to be in a separate method for clarity and testability.

Reply:
`"/"` → `os.sep`

Reply:
+1 on testability comment.  Implies unit tests should be written.